pkg/asset/machines/master.go

```diff
@@ -19,6 +19,8 @@ import (
 	libvirtprovider "github.com/openshift/cluster-api-provider-libvirt/pkg/apis/libvirtproviderconfig/v1beta1"
 	ovirtproviderapi "github.com/openshift/cluster-api-provider-ovirt/pkg/apis"
 	ovirtprovider "github.com/openshift/cluster-api-provider-ovirt/pkg/apis/ovirtprovider/v1beta1"
+	nutanixapi "github.com/openshift/machine-api-provider-nutanix/pkg/apis"
+	nutanixprovider "github.com/openshift/machine-api-provider-nutanix/pkg/apis/nutanixprovider/v1beta1"
```

Comment:
Can we get these API types from github.com/openshift/api/machine/v1 instead? That keeps the installer from having to pull in dependencies like github.com/terraform-providers/terraform-provider-nutanix.

Reply:
@staebler We will be doing that in a separate PR. Yanhua from our side is currently working on that PR.

Reply:
In general, our perspective on this is that let’s get the existing stack merged. Some of the changes that require moving the dependency source over from MAPI Provider to API are better done all at once in a subsequent PR.

Reply:
Help me to understand the difference between the types in openshift/api and openshift/machine-api-provider-nutanix, currently. Is what is in openshift/api what we are going to use for the release? Are we blocked by the changes that need to be made to the nutanix machine-api operator to support using the types in openshift/api?

Reply:
@staebler AFAIK, the machine types are being ported from MAPI provider to API by Yanhua.